ci/lint.sh

```diff
@@ -91,6 +91,15 @@ if [ "$LINT" ]; then
     fi
     echo "Check for invalid testing DONE"
 
+    # Check for imports from pandas.core.common instead
+    # of `import pandas.core.common as com`
+    echo "Check for non-standard imports"
+    grep -R --include="*.py*" -E "from pandas.core.common import " pandas
```

Comment:
this rule includes too many things, you want to *only* accept ``from pandas.core import common as com``. anything else involving ``pandas.core.common`` should error

Reply:
This is tough to do with grep since there are things like `sys.modules['pandas.core.common']`, `warnings.warn("pandas.core.common.{t} is deprecated`, `# the pandas.core.common introspectors`.  Even if I string together a workable grep command, maintaining it will be a PITA.  I think the 1-line 98% solution is the way to go on this one.